website/docs/core-concepts/formats.md

```diff
@@ -4,4 +4,23 @@ sidebar_position: 5
 
 # Formats
 
-## Coming Soon!
\ No newline at end of file
+Formats for a space can be created from the Formats menu. Default formats for Dega can be added by clicking on the Create default Formats. 
+
+The default formats for Dega are
+- Fact-Check
+- Article
+
+Fact-Check and Article formats are mandatory formats for Dega.
+
+- Name: This allows the user to add a Name of the format
+- Featured: Check if the claimant is featured
```

Comment:
This field can be used to `highlight` the `Claimant` on the website if `featured` is enabled.

Reply:
updated